backend/src/apis/snapshot.rs

```diff
@@ -280,8 +262,13 @@ mod tests {
             "0x220b71671b649c03714da9c621285943f3cbcdc6".to_string(), // ERC20 token address
         ];
 
-        let calldata = snapshot
-            .generate_proof_of_solvency(asset_addresses.clone(), None)
+        // In this test, we assume that the balances of the accounts in the snapshot are 556863 for both assets
```

Comment:
Can we remove the `ignore` for this test?

Reply:
Removed at 9ada335